[2022-06-27 13:33:13,192] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_B scheduled__2022-06-08T00:00:00+00:00 [queued]>
[2022-06-27 13:33:13,225] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_B scheduled__2022-06-08T00:00:00+00:00 [queued]>
[2022-06-27 13:33:13,227] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:33:13,228] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:33:13,229] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:33:13,291] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_B> on 2022-06-08 00:00:00+00:00
[2022-06-27 13:33:13,321] {standard_task_runner.py:52} INFO - Started process 58421 to run task
[2022-06-27 13:33:13,350] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_B', 'scheduled__2022-06-08T00:00:00+00:00', '--job-id', '794', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp8475l7h9', '--error-file', '/tmp/tmp7vsxhr94']
[2022-06-27 13:33:13,358] {standard_task_runner.py:80} INFO - Job 794: Subtask model_B
[2022-06-27 13:33:13,581] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_B scheduled__2022-06-08T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:33:13,840] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_B
AIRFLOW_CTX_EXECUTION_DATE=2022-06-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-08T00:00:00+00:00
[2022-06-27 13:33:13,844] {python.py:173} INFO - Done. Returned value was: 10
[2022-06-27 13:33:13,921] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_B, execution_date=20220608T000000, start_date=20220627T133313, end_date=20220627T133313
[2022-06-27 13:33:14,001] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:33:14,145] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
